.go-version

```diff
@@ -0,0 +1 @@
+1.20.7
```

Comment:
Can we move this file under the `server` directory?

Reply:
I think that'd depend on if there's any tooling that requires it to be at the root like with `.nvmrc`

Reply:
AFAIK, there isn't anything like that. This file is equivalent to go.mod, and will eventually be replaced by that in Go 1.21. So it should be at the same level as that.